err_stash.py

```diff
@@ -387,7 +387,7 @@ def make_pr_link(url, project, slug, from_branch, to_branch):
     """Generates a URL that can be used to create a PR"""
     from urllib.parse import urlencode
 
-    if "github.com/" in url:
+    if "github.com" in url:
         result = "{url}/{organization}/{repo_name}/compare/{from_branch}".format(
```

Comment:
Since you are changing this and you already have the `target_branch` in hand, you could generate the url with both source/target instead of only source (which makes more sense than always using master as target).

Syntax is f'{url}/{organization}/{repo_name}/compare/{to_branch}...{from_branch}'

Reply:
Fixed, thanks for the heads up!